plugins/s3_auth/Makefile.inc

```diff
@@ -16,3 +16,12 @@
 
 pkglib_LTLIBRARIES += s3_auth/s3_auth.la
 s3_auth_s3_auth_la_SOURCES = s3_auth/s3_auth.cc
+
+#GOOGLETEST_DIR=/opt/gtest
```

Comment:
@gtenev I have no objection to bringing this in.  It looks very interesting to me and I'm not looking at using goolge test myself.